packages/firestore/CONTRIBUTING.md

```diff
@@ -16,10 +18,30 @@ See
 [Automated Setup](https://github.com/firebase/firebase-js-sdk#automated-setup) 
 for more details.
 
+### Setting up the Firestore emulator
+
+Running the integration tests against the Firestore emulator is much faster
+than running against the prod server. The integration tests require that the
+Firestore emulator is running on port 8080, which is default when running it
+via CLI.
+
+  * [Install the Firebase CLI](https://firebase.google.com/docs/cli/).
+    ```
+    npm install -g firebase-tools
+    ```
+  * [Install the Firestore
+    emulator](https://firebase.google.com/docs/firestore/security/test-rules-emulator#install_the_emulator).
+    ```
+    firebase setup:emulators:firestore
+    ```
+  * Run the emulator
+    ```
+    firebase serve --only firestore
+    ```
 
-## Running Firestore Tests
+### Running Integration Tests
```

Comment:
We cover both unit and integration tests below so not sure "Integration" is appropriate.